docs/api.md

```diff
@@ -1016,6 +1019,10 @@ event instead.
 
 Emitted when the server changes any of the game properties.
 
+#### "resourcePack"
```

Comment:
Should indicate that it also emits data from the resource pack

Reply:
right